### DISCUSSION TITLE
Sleep behaviour after G Helper in temps

### DISCUSSION THREAD

**Discussion by AylakDemir** [seerge/g-helper#3118] [Q&A]
Hello, 
first of all thanks for all the work I just started using the program yesterday and it's pretty neat! 
Today after all the updates and restarting the pc, I've noticed that the CPU temp gets high especially when it's on sleep. After waking, the fans kick in and bring the temperature lower but somehow they seem to not work in sleep. 
Another issue: I've stopped the Asus stuff including MyAsus. Then set the charging limit to 85% yesterday. Today I found it at 100% and it slowly got back to 85%. What should I check/change?
I followed your guide and didn't play with the fans + power settings so far. I also did as suggested revert back to factory settings of balanced power mode in windows settings. I have the 2021 G15. Thanks again!

**Comment by seerge** [comment]
@AylakDemir hello :)

Sleep is windows functionality, G-Helper is not really able to interact with that.

When laptop is properly sleeping (on hardware level), windows suspends all processes and fans should stop. So that's a normal behavior :) When it wakes up, it's logically that some CPU intensive tasks start so fans will probably kick in as well. Fans are controlled by BIOS based on actual temps.

For charge limit : if laptop is already charged to 100%, it won't discharge to 85% by itself, unless you unplug. Charge limiter usually only stops charging above set limit.

**Comment by AylakDemir** [comment]
@seerge Thank you for your answer! I understand what you're saying but this wasn't the case before I installed G-helper and deleted AC. Now I prevent the laptop to go sleep when it's on power but CPU tends to pike every now and then when I leave it idle and gets to 80+ degrees which makes the fans kick in. Do you think I did something wrong during the installation process?

**Reply by seerge** [reply]
G-Helper doesn't have any installation process :) It's just an exe that you run (or you don't and it has 0 effect on the system).

Did you use official AC uninstaller app?

Could it be that windows runs some tasks now in background that don't let it sleep? Try to [run sleepstudy](https://learn.microsoft.com/en-us/windows-hardware/design/device-experiences/modern-standby-sleepstudy) to see what exactly is happening during this "sleep"
